### ISSUE TITLE
[feature request] summary options/formatting

### ISSUE THREAD

**Issue by WHiZSTA** [medartus/tldr-it-extension#1]
Can we get some summary options like how long of a summary we want, as the summaries are quite short and many times don't provide a good picture of the overall content, and also formatting options like bullet points, as the one big blob of text makes it hard to follow. It currently even strips out extra spacing added manually to submitted summaries.

A good visual example of what i'm referring to might be:
[](https://chrome.google.com/webstore/detail/tldr-this-free-automatic/jjmnbidkmmabkalampbilajnheeokpoj/)

Thanks!

**Comment by medartus** [comment]
Hi,

I made modification on the extension and the server to auto indent content:
- Old and new content with indentation will be rendered as expected with multiple lines
- New auto-summarized content will be automatically indented based on the multiple paragraphs created

> Please update to version 1.0.1 to get latest UI changes!

Regarding the choice of the size of the summary, I can't help you much since I do not have control over the content length. Most of the time if summaries are short, it's mainly due to short content to summarize in the first place.

Thanks for using TLDR; it ! 😄
